addons/l10n_at/data/account_tax_template.xml

```diff
@@ -5,24 +5,186 @@
+                  'minus_report_line_ids': [ref('tax_report_line_l10n_at_tva_line_4_1'),
+                  ref('tax_report_line_l10n_at_tva_line_4_3')],
+                }),
+
+                (0,0,{
+                  'factor_percent': 100,
+                  'repartition_type': 'tax',
+                  'account_id': ref('chart_at_template_3505'),
+                  'minus_report_line_ids': [ref('tax_report_line_l10n_at_tva_line_4_3')],
+                }),
+            ]"/>
+        </record>
+        <record id="account_tax_template_sales_rev_charge_0_code021_1a" model="account.tax.template">
+            <field name="name">UST_021 § 19 Abs. 1a (Steuerschuld betrifft Leistungsempfänger)</field>
+            <field name="description">0%</field>
+            <field name="chart_template_id" ref="l10n_at_chart_template" />
+            <field name="sequence">400</field>
+            <field name="type_tax_use">sale</field>
+            <field eval="0.0" name="amount" />
+            <field name="amount_type">percent</field>
+            <field name="price_include" eval="0" />
+            <field name="tax_group_id" ref="tax_group_0" />
+            <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0,{
+                  'factor_percent': 100,
+                  'repartition_type': 'base',
+                  'plus_report_line_ids': [ref('tax_report_line_l10n_at_tva_line_4_1'),
+                  ref('tax_report_line_l10n_at_tva_line_4_3')],
+                }),
+                (0,0,{
+                  'factor_percent': 100,
+                  'repartition_type': 'tax',
+                  'account_id': ref('chart_at_template_3505'),
+                  'plus_report_line_ids': [ref('tax_report_line_l10n_at_tva_line_4_3')],
+                }),
+            ]"/>
+            <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0,{
+                  'factor_percent': 100,
+                  'repartition_type': 'base',
+                  'minus_report_line_ids': [ref('tax_report_line_l10n_at_tva_line_4_1'),
+                  ref('tax_report_line_l10n_at_tva_line_4_3')],
+                }),
+
+                (0,0,{
+                  'factor_percent': 100,
+                  'repartition_type': 'tax',
+                  'account_id': ref('chart_at_template_3505'),
+                  'minus_report_line_ids': [ref('tax_report_line_l10n_at_tva_line_4_3')],
+                }),
+            ]"/>
+        </record>
+        <record id="account_tax_template_sales_rev_charge_0_code021_1b" model="account.tax.template">
+            <field name="name">UST_021 § 19 Abs. 1b (Steuerschuld betrifft Leistungsempfänger)</field>
+            <field name="description">0%</field>
+            <field name="chart_template_id" ref="l10n_at_chart_template" />
+            <field name="sequence">400</field>
+            <field name="type_tax_use">sale</field>
+            <field eval="0.0" name="amount" />
```

Comment:
Not sure I can follow, but yeah, probably it has grown over time instead of doing them all at once...but at least, this is quite close to real life work on the product 😉 but yeah, I guess we will have more things being changed (XML -> CSV, refactoring of framework # 100, etc.)